libs/database/src/lib/dtos/ads.dto.ts

```diff
@@ -89,3 +96,11 @@ export class AdsCampaignStatisticResponse {
   };
   'CPM': number;
 }
+export class adsQuery extends PaginationQuery {
```

Comment:
```suggestion
export class AdsQuery extends PaginationQuery {
```

ชื่อ class/type ใช้ `PascalCase` ส่วน `camelCase` เอาไว้พวก variables